Lib/test/test_unittest/testmock/testmock.py

```diff
@@ -129,6 +129,11 @@ def test_create_autospec_should_be_configurable_by_kwargs(self):
         # pass kwargs with respect to the parent mock.
         self.assertEqual(class_mock().return_value.meth.side_effect, None)
 
+    def test_create_autospec_correctly_handles_name(self):
```

Comment:
Instead of creating a new test case, can you modify this existing one to accommodate for `spec_set`?

https://github.com/python/cpython/blob/455af0922c454a05d7116db485ddb4f28502b7f2/Lib/test/test_unittest/testmock/testmock.py#L1727-L1730

```diff
def test_create_autospec_with_name(self):
-    m = mock.create_autospec(object(), name='sweet_func')
+    m = mock.create_autospec(object(), spec_set=True, name='sweet_func')
    self.assertIn('sweet_func', repr(m))
```

Reply:
I prefer to have two tests over just one: with `spec_set=True` and without :)